internal/engine/k8swatch/event_watch_manager.go

```diff
@@ -191,7 +193,7 @@ func (m *EventWatchManager) dispatchEventsLoop(ctx context.Context, ch <-chan *v
 			// TODO(nick): We might need to remove this check and optimize
 			// it in a different way. We want Tilt to be to attach to existing
 			// resources, and these resources might have pre-existing events.
-			if event.ObjectMeta.CreationTimestamp.Time.Before(tiltStartTime) {
+			if !timecmp.AfterOrEqual(event.ObjectMeta.CreationTimestamp, tiltStartTime) {
```

Comment:
Realistically, this logic was fine before - you could pretty safely ignore any events that happened the same second that Tilt started. But the stricter construction meant some tests failed, and since they use wall-clock, this was cleaner (and literally faster) than making them `time.Sleep(time.Second)`.